test/test_plugins_lhcb.py

```diff
@@ -0,0 +1,380 @@
+"""
+Tests for the LHCb experiment metadata plugins.
```

Comment:
Minor detail: Here we could also rename `metadata`. Same for the other test files.

Reply:
Good catch! Done.